sdk/src/main/java/io/customer/sdk/util/StaticSettingsProvider.kt

```diff
@@ -1,6 +1,6 @@
 package io.customer.sdk.util
 
-import io.customer.sdk.BuildConfig
+import io.customer.tracking.BuildConfig
```

Comment:
Yes, the reason is linked. This again is due to difference in naming. I think Android Gradle now uses namespace instead of its path which resulted in change in BuildConfig file. Also, this is internal and should not have any impact on customers.